tests/darwin/dataset/split_manager_test.py

```diff
@@ -15,7 +16,15 @@ def test_requires_scikit_learn():
 
 
 def describe_classification_dataset():
-    @pytest.mark.parametrize("val_percentage,test_percentage", [(0.2, 0.3), (0.3, 0.2), (0.2, 0)])
+    @pytest.mark.parametrize("val_percentage,test_percentage", [(0, 0.3), (0, 0), (0.2, 0), (0.5, 0.5), (1, 0.1)])
+    def it_raises_for_invalid_split_configuration(
+        team_slug: str, team_extracted_dataset_path: Path, val_percentage: float, test_percentage: float
+    ):
+        with pytest.raises(ValueError):
+            root = team_extracted_dataset_path / team_slug / "sl"
+            split_dataset(root, release_name="latest", val_percentage=val_percentage, test_percentage=test_percentage)
+
+    @pytest.mark.parametrize("val_percentage,test_percentage", [(0.2, 0.3), (0.3, 0.2)])
     def it_should_split_a_dataset(
```

Comment:
Does this test a random or stratified split?

Reply:
That should do both!